src/app/results/results.module.ts

```diff
@@ -0,0 +1,29 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {ResultsComponent} from './results.component';
+import {ResultsItemComponent} from './results-item/results-item.component';
+import {AddButtonComponent} from '../shared/ui/add-button/add-button.component';
+import {ResultsDataService} from '../services/results.data.service';
+
+@NgModule({
+    declarations: [
+        ResultsComponent,
+        ResultsItemComponent,
+        AddButtonComponent
+    ],
```

Comment:
I prefer to create modules on logical parts of the application. It is not necessary to be a hole page. In this case I just consider, that results and goals should be a modules, because it could be separated and used on different pages in future.